src/screens/WriteExerciseScreen.tsx

```diff
@@ -113,7 +97,7 @@ const WriteExerciseScreen = ({ navigation, route }: WriteExerciseScreenPropsType
           </TouchableOpacity>
         ),
         headerRight: () => (
-          <Text style={styles.headerText}>{`${currentDocumentNumber + 1} of ${documents.length}`}</Text>
+          <Text style={styles.headerText}>{`${currentDocumentNumber + 1} of ${documents?.length}`}</Text>
```

Comment:
Is this displayed if there are no documents as well? If so, it probably says something like `1 of undefined`

Reply:
if shows `1 of 0`, happens while loading is not finished, think this behaviour is oke for now.